omero/sysadmins/server-permissions.rst

```diff
@@ -36,17 +36,21 @@ Users
     **Group member**
         This is the standard user.
 
-
```

Comment:
this has not been handled in this PR. Nevertheless the intro bit has at the end an untrue, misleading sentence which I think should be fixed.
https://github.com/openmicroscopy/ome-documentation/pull/1778/commits/c515595e2e351e9f69869b4038e0f0b0441f74c6#diff-3c1d6113c0b3e790090e598763e385caR17

``...easily copy their data into another group.`` -> ``...easily move their data into another group.``

Reply:
Again, the same as above comment, in the "old" text there is
https://github.com/openmicroscopy/ome-documentation/pull/1778/commits/c515595e2e351e9f69869b4038e0f0b0441f74c6#diff-3c1d6113c0b3e790090e598763e385caR32
I think the formulation ``owner has some additional rights... than a standard group member`` should be reformulated, this does not seem a good english (``additional .... than``)

Reply:
Pushed two fixes for these